prism/src/main/java/org/apache/falcon/jdbc/MonitoringJdbcStateStore.java

```diff
@@ -48,7 +48,7 @@ public void putMonitoredEntity(String entityName, String entityType) throws Falc
 
         MonitoredEntityBean monitoredEntityBean = new MonitoredEntityBean();
         monitoredEntityBean.setEntityName(entityName);
-        monitoredEntityBean.setEntityType(entityType);
+        monitoredEntityBean.setEntityType(entityType.toLowerCase());
```

Comment:
By Default we were storing in capitals but in case of api call we are passing entityType whatever user provides i.e process,Process or PROCESS in case the db is case sensitive we will not get any results because of the where clause in the SQL.

As entityType is internal to falcon so I thought of storing and querying it in lowercase.Let me know your thoughts on this.